Cargo.toml

```diff
@@ -332,7 +333,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "=1.0.128", features = ["arbitrary_precision"] }
 similar-asserts = "1.6"
 soldeer-commands = "=0.5.2"
-strum = "0.27"
+strum = "0.26"
```

Comment:
different from upstream?

Reply:
We cannot move to `0.27` as that is the breaking change introduced that is also included on alloy-chains. Probably should add a note here.